src/tests/end-to-end/common/browser.ts

```diff
@@ -137,6 +137,18 @@ export class Browser {
         return { detailsViewPage, targetPage };
     }
 
+    public async newQuickAssess(
```

Comment:
nit: it's not a lot of code necessarily but might make sense to de-duplicate this with the assessment method.